src/BooleanParser.php

```diff
@@ -4,6 +4,11 @@
 
 class BooleanParser extends AbstractValueParser
 {
+    protected function describe()
+    {
+        return "either true or false";
```

Comment:
`return "either true, false, 1 or 0";` might be the complete description, but I guess it doesn't really matter :)